ckanext/custom_schema/schemas/dataset.yaml

```diff
@@ -142,6 +131,50 @@ dataset_fields:
     value:  R/P2Y
   - label: Decennial
     value:  R/P10Y
+    
+## "temporalNotes"
+## This field needs revisiting regarding the free form nature of the text box vs. dropdown
+## Start and end date for the data in the dataset
+- field_name: temporal
+  label: Temporal Coverage
+  form_snippet: markdown.html
+
+## "granularity"
+## This field needs revisiting regarding the free form nature of the text box vs. dropdown
+- field_name: granularity
+  label: Granularity
+  form_placeholder: eg. county, census track, monthly, daily
```

Comment:
Should the examples in the form placeholder be title case or lowercase. I ask because the Geographic Coverage field has an example in title case.